SPDY/SPDYStreamManager.m

```diff
@@ -21,8 +21,8 @@ @interface SPDYStreamNode : NSObject
 @implementation SPDYStreamNode
 {
   @public
-    __strong SPDYStreamNode *next;
-    __strong SPDYStreamNode *prev;
+    __weak SPDYStreamNode *next;
+    __weak SPDYStreamNode *prev;
```

Comment:
Generally, with linked lists...you want a weak ref to the previous node and a strong ref to the next node.  That way when the head node is released it will cascade down.  Otherwise, you need some additional container for maintaining references.